cmd/updater/command/command.go

```diff
@@ -53,8 +54,8 @@ Datadog Updater updates your agents based on requests received from the Datadog
 		SilenceUsage: true,
 	}
 
-	agentCmd.PersistentFlags().StringVarP(&globalParams.ConfFilePath, "cfgpath", "c", "", "path to directory containing updater.yaml")
-	agentCmd.PersistentFlags().StringVarP(&globalParams.Package, "package", "p", "", "package to update")
+	agentCmd.PersistentFlags().StringVarP(&globalParams.ConfFilePath, "cfgpath", "c", pkgconfig.DefaultUpdaterLogFile, "path to directory containing updater.yaml")
```

Comment:
The description is `path to directory containing updater.yaml` but the default value is a path to a file (not directory !) named `updater.log`, is that expected ?